### PR TITLE
fix for issue 286

### PR DESCRIPTION
**Pull request by dafnapension**
a simple fix..  perhaps good enough for now.

### CODE REVIEW

**Comment by perlitz** [comment]
Hi @dafnapension!
nice to git-meet :) 
you will need to run `pre-commit install` inside the project to make the formatters work when you commit, this way pass the you will pass the Code Quality & Security / verify-pre-commit-executed (pull_request) tests

File: src/unitxt/loaders.py [hunk 1]
```diff
@@ -39,13 +39,19 @@ class LoadHF(Loader):
     cached = False
 
     def process(self):
-        dataset = hf_load_dataset(
-            self.path, name=self.name, data_dir=self.data_dir, data_files=self.data_files, streaming=self.streaming
-        )
+        try:
```
Comment (perlitz) [inline src/unitxt/loaders.py:42]: can't you just LoadHF with streaming=False?
and

if not streaming:
    for split in dataset.keys():
        dataset[split] = dataset[split].to_iterable_dataset()

?

**Comment by perlitz** [comment]
Also, you will have to add test to improve the coverage

**Comment by dafnapension** [comment]
Thank you, @perlitz ,
Can you please point for me, novice, to the unittest that covers **any** line from LoadHF?

File: src/unitxt/loaders.py (hunk 1 above)
Comment (yoavkatz) [reply src/unitxt/loaders.py:42]: I think you can do as you suggest:
if (self.streaming):
    dataset = hf_load_dataset(
                self.path, name=self.name, data_dir=self.data_dir, data_files=self.data_files, streaming=self.streaming
            )
else:
      dataset = hf_load_dataset(
                self.path, name=self.name, data_dir=self.data_dir, data_files=self.data_files, streaming=False
            )
            for split in dataset.keys():
                dataset[split] = dataset[split].to_iterable_dataset()
                
Then the card creator , can have specify  

loader = LoadHF(path="GEM/xlsum", name="igbo", streaming=False) 


If the error message when stream = True and reading zip files is not clear, you can catch the exception, and raise a new one. 


In any case, best to have a test that checks that the exceptions is generated.

**Review by yoavkatz — CHANGES_REQUESTED** [review]
Hi Dafna, please see the conversation.

File: src/unitxt/loaders.py (hunk 1 above)
Comment (dafnapension) [reply src/unitxt/loaders.py:42]: Thank you, @yoavkatz , I do cover the exception catch in the test. I see these lines are all green in the coverage report.

Comment (dafnapension) [reply src/unitxt/loaders.py:42]: @yoavkatz , can you, then, approve this PR?

**Review by yoavkatz — APPROVED** [review]

**Comment by yoavkatz** [comment]
Closing to add sign to solve DCO issue.

**Pull request closed without merging**